client/components/FirebaseImg.tsx

```diff
@@ -38,8 +38,8 @@ const FirebaseImg = (props) => {
             style={[
                 styles.bikeItemImage,
                 {
-                    height: props.height || 85,
-                    width: props.width || 60
+                    height: props.height || imgProps.height || 85,
```

Comment:
I think this way is messy. I would prefer to go back to using the FirebaseImgProps type where height and width are required and the rest are optional. We could potentially make all of the style params just a style that is included as a prop instead of each individual style.

```
export type FirebaseImgProps = {
    img_id: string, 
    width: string | number | undefined,
    height: string | number | undefined,
    borderRadius?: number,
    marginRight?: number,
    aspectRatio?: string | number
}
```